### DISCUSSION TITLE
How to rename lego hub running pybricks

### DISCUSSION THREAD

**Discussion by CTFlorian** [pybricks/support#775] [Q&A] [answered]
I have several hubs ALL WITH THE SAME NAME.  
How do i change it.

**Comment by dlech** [comment, marked as answer]
You have to set the name when you flash the firmware. At http://code.pybricks.com you enter the name in the settings: 

<img width="366" alt="image" src="https://user-images.githubusercontent.com/963645/199830210-2e3769ed-c480-4bd9-ac56-355220caa0ec.png">

At https://beta.pybricks.com, it is part of the firmware install dialog.
<img width="462" alt="image" src="https://user-images.githubusercontent.com/963645/199830427-9cbcccb8-872a-4c2d-8c9d-3aed66d97ce1.png">
